src/pages/resultsView/coExpression/CoExpressionViz.tsx

```diff
@@ -262,18 +276,21 @@ export default class CoExpressionViz extends React.Component<ICoExpressionVizPro
     }
 
     @computed get showLogScaleControls() {
-        const profileId = this.props.molecularProfile.molecularProfileId.toLowerCase();
+        const profileXId = this.props.profileX.molecularProfileId.toLowerCase();
+        const profileYId = this.props.profileY.molecularProfileId.toLowerCase();
         return (
-            (profileId.indexOf("rna_seq") > -1) &&
-            (profileId.indexOf("zscore") === -1)
+            ((profileXId.indexOf("rna_seq") > -1) &&
```

Comment:
we have a helper method that does this called `logScalePossible`

Reply:
`logScalePossible` is from the Plots tab and requires an element of type `AxisMenuSelection`, I don't think it can be reused here.

Reply:
good point. but I really don't want to duplicate this logic again. could you please add a util function somewhere, might as well just put it in PlotsTabUtils, like
```
    export function logScalePossibleForProfile(profileId:string) {
        return !(/zscore/i.test(profileId)) &&
            /rna_seq/i.test(profileId);
    }
```
And use that here, and in logScalePossible so it would be `...&& logScalePossibleForProfile(axisSelection.dataSourceId)`?